keps/sig-node/1769-memory-manager/README.md

```diff
@@ -778,3 +780,19 @@ The Memory Manager sets and enforces cgroup memory limit for ("on behalf of") a
 [hugepage-issue]: https://github.com/kubernetes/kubernetes/issues/80716
 [memory-issue]: https://github.com/kubernetes/kubernetes/issues/81009
 
+### Windows considerations
+
+Numa nodes can not be guaranteed via the Windows API, instead an [ideal Numa](https://learn.microsoft.com/en-us/windows/win32/procthread/numa-support#numa-support-on-systems-with-more-than-64-logical-processors) node can be 
+configured via the [PROC_THREAD_ATTRIBUTE_PREFERRED_NODE](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-updateprocthreadattribute).  
+Using Memory manager's internal mapping this should provide the desired behavior in most cases. It is possible that a CPU could access memory from a different Numa Node than it is currently in, resulting in decreased performance.  For this reason, 
+we will add documentation in addition to a log warning message in kubelet to help raise awareness.
+If state is undesirable then `single-numa-node` and the CPU manager should be configured in the Topology Manager policy setting 
```

Comment:
"If state" here doesn't read correctly gramatically. "If this state" ?

Reply:
updated wording for clarity